### PR TITLE
create Product app, add ProductCategory model

### PR DESCRIPTION
**Pull request by meeklife**

### CODE REVIEW

File: apps/products/models.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+from apps.common import models as base_models
+
+
+class ProductCategory(base_models.BaseModel):
+    name = models.CharField(max_length=50)
+    description = models.TextField()
```
Comment (kryzbone) [inline apps/products/models.py:8]: @meeklife make the description field optional

File: config/api_urls.py [hunk 2]
```diff
@@ -5,4 +5,5 @@
 urlpatterns = [
     path("brands/", include("apps.brands.urls")),
     path("", include("apps.users.urls")),
```
Comment (kryzbone) [inline config/api_urls.py:7]: as a conversion always make sure the path with "" the last on the urlpatterns list

Comment (meeklife) [reply config/api_urls.py:7]: > as a conversion always make sure the path with "" the last on the urlpatterns list

ohk
noted

Comment (meeklife) [reply config/api_urls.py:7]: > as a conversion always make sure the path with "" the last on the urlpatterns list

will make the changes right away

**Review by kryzbone — COMMENTED** [review]
Approved

**Pull request merged**